packages/docs/docs/api-reference/utility-functions.md

```diff
@@ -281,6 +281,21 @@ Returns `undefined` when a valid discriminator is not present.
 
 - string | undefined: The `discriminator.propertyName` if it exists in the schema, otherwise `undefined`
 
+### getDateElementProps( date: DateObject, time: boolean, yearRange: [number, number] = [1900, new Date().getFullYear() + 2], format: DateElementFormat = 'YMD')
```

Comment:
We don't need to mention the parameters here as they are explicitly mentioned below
```suggestion
### getDateElementProps()
```